lib/pull.ml

```diff
@@ -50,6 +50,48 @@ let pre_pull_clean_up ~full ~duniverse_dir duniverse =
         Bos.OS.Dir.delete ~must_exist:false ~recurse:true
           Fpath.(duniverse_dir / dir))
 
+let duniverse_documentation =
+  {|\
+duniverse
+=========
```

Comment:
Propbably better to use the `# duniverse` syntax here!

Reply:
Good point, I've changed this while rebasing :+1: